skada/_dasvm.py

```diff
@@ -0,0 +1,225 @@
+# Author: Ruben Bueno <ruben.bueno@polytechnique.edu>
+# dasvm implementation
+
+import numpy as np
+import math
+
+from skada.base import BaseEstimator
+from skada.utils import check_X_y_domain, source_target_split
+from sklearn.base import clone
+
+from sklearn.svm import SVC
+
+
+class DASVMEstimator(BaseEstimator):
+    """
+    Simple dasvm estimator
+    """
+
+    __metadata_request__fit = {'sample_domain': True}
+    __metadata_request__transform = {'sample_domain': True, 'allow_source': True}
+
+    def __init__(
+            self, base_estimator=None,
```

Comment:
To be more consistent with the rest of the library, can you put one parameters per line